app/models/BulkCalculationRequest.scala

```diff
@@ -106,9 +108,9 @@ case class ProcessReadyCalculationRequest(bulkId: String,
 
 object ProcessReadyCalculationRequest {
   // $COVERAGE-OFF$
+  implicit val dateFrmat = MongoJavatimeFormats.localDateFormat
```

Comment:
would it be difficult to correct this spelling error?

Reply:
nopes, i can fix that